src/foam/net/web/HTTPRequest.js

```diff
@@ -157,7 +157,10 @@ foam.CLASS({
         // Use Promise.reject so crappy debuggers don't pause here
         // throw resp;
         return Promise.reject(resp);
-      }.bind(this));
+      }.bind(this))
```

Comment:
Instead of doing .bind(this) (I know you didn't write the code, but while you're here) you could just switch to => notation which would do the same.

Reply:
good catch - appreciate - and done